apps/condo/domains/common/utils/FiltersStorage.ts

```diff
@@ -0,0 +1,27 @@
+import pickBy from 'lodash/pickBy'
+import isString from 'lodash/isString'
+import { updateQuery } from './filters.utils'
+import { NextRouter } from 'next/router'
+import { FiltersFromQueryType } from './tables.utils'
+
+export enum FILTER_TABLE_KEYS {
+    TICKET = 'ticketsFilter',
+}
+
+export class FiltersStorage {
+    public static saveFilters (tableKey: FILTER_TABLE_KEYS, filters: FiltersFromQueryType | string): void {
+        localStorage.setItem(tableKey, JSON.stringify(pickBy(filters)))
+    }
+
+    public static async loadFilters (tableKey: FILTER_TABLE_KEYS, router: NextRouter): Promise<void> {
+        const filters = localStorage.getItem(tableKey)
+
+        if (isString(filters)) {
```

Comment:
You should check whether it is json, not only string. Wrap it to try-catch block